homeassistant/components/azure_storage/__init__.py

```diff
@@ -0,0 +1,78 @@
+"""The Azure Storage integration."""
+
+from azure.core.exceptions import (
+    ClientAuthenticationError,
+    HttpResponseError,
+    ResourceNotFoundError,
+)
+from azure.core.pipeline.transport import (  # pylint: disable=no-name-in-module
+    AioHttpTransport,
+)
```

Comment:
?

Reply:
ah yeah. No idea why, but pylint (and VSCode for that matter) can't find it. No idea why. It's there and works.